normandy/recipes/filters.py

```diff
@@ -592,6 +592,50 @@ def to_jexl(self):
         return f"(normandy.os.isWindows && {super().to_jexl()})"
 
 
+class WindowsVersionFilter(BaseComparisonFilter):
+    """
+    Match a user based on what windows version they are running. This filter
+    creates jexl that compares the windows NT version.
+
+    .. attribute:: type
+
+        ``windows_version``
+
+    .. attribute:: value
+        string, must be one of the following: 6.1, 6.2, 6.3, 10.0
```

Comment:
Shouldn't this be a number, not a string?

Reply:
ok 👍